### DISCUSSION TITLE
lazyvim Not valid In Snap

### DISCUSSION THREAD

**Discussion by huifer** [LazyVim/LazyVim#228] [Q&A]
I used snap to install neovim and then downloaded lazyVim and ran nvim but it didn't work。

```
[root@system config]# nvim -version
NVIM v0.9.0-dev-613+gef18c9f9b
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/build/nvim/parts/nvim/build/build/cmake.config -I/build/nvim/parts/nvim/build/src -I/build/nvim/parts/nvim/build/.deps/usr/include -I/usr/include -I/build/nvim/parts/nvim/build/build/src/nvim/auto -I/build/nvim/parts/nvim/build/build/include
Compiled by root@lcy02-amd64-005

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

```

**Comment by folke** [comment]
Would be great to show what exactly didnt work?

**Comment by huifer** [comment]
When I execute the nvim command I go directly to the screen as shown in the figure
![image](https://user-images.githubusercontent.com/26766909/218123667-b39b020d-f936-41b4-8459-00a8ff339b3a.png)  @folke

**Reply by folke** [reply]
That means you didn't install LazyVim. Unless the Snap version doesn't read your `~/.config/nvim`?
Either way, this is not a LazyVim issue as far as I can tell.

**Reply by huifer** [reply]
> That means you didn't install LazyVim. Unless the Snap version doesn't read your `~/.config/nvim`? Either way, this is not a LazyVim issue as far as I can tell.

How can I make Snap read  `~/.config/nvim`

**Reply by folke** [reply]
I have no idea. I don't use snap. As far as I understand, snap containerizes Neovim, so not sure that can even work.

Maybe check the snap docs?

**Reply by huifer** [reply]
Thank you

**Comment by ser** [comment]
neovim snap is broken for root, but easily fixable:

```
ln -s /root /home
```

https://github.com/neovim/neovim-snap/issues/46
